utils/nctl/sh/scenarios/client.sh

```diff
@@ -1317,11 +1315,12 @@ function get_generate_completion_shells() {
 function get_client_subcommand_list() {
     local OUTPUT
     OUTPUT=$($(get_path_to_client) --help \
-        | grep -A 999 SUBCOMMANDS \
-        | awk -F'    ' '{print $2}' \
+        | awk -v RS= 'NR==3' \
+        | sed -e 's/^[ \t]*//' \
```

Comment:
The latest client code in `release-2.0.0` soft breaks the help lines at 100 chars if the terminal width can't be determined.  So it seems like this change below strips such lines from the output, as they all have more than 2 leading spaces:
```suggestion
        | sed -e '/^[ \t]\{3,\}/d' \
```

Reply:
updated